ginga/examples/layouts/twofer/README.md

```diff
@@ -0,0 +1,14 @@
+### About
+
+This directory defines a set of files that provide a riff on the standard
+look and feel of the reference viewer layout called "twofer".  It provides
+a second large pane on the bottom to take some of the plugins that would
```

Comment:
Maybe given an example or two of which plugin would utilize the second large pane? I tried `Histogram` but does not seem like that plugin uses it.

Reply:
@pllim, see the `screenshot.png` in the `twofer` directory.  The plugins that will open there are `Pan`, `Info`, `Errors`, `Command`, `MultiDim`, `RC`, `SAMP`, `FBrowser` ([G]) and `Zoom`.

Reply:
> Or is ~ expansion not supported?

I think that is a host shell issue, but I am not sure. Was this on Linux? Normally `bash` expands those when issued from the command line, so the program invoked doesn't see that unless it is dealing with a path typed directly into an input opened by the program. Not sure how that handled on Windows.

Reply:
> Was this on Linux?

Yes. I don't think Windows understand `~` natively.